pkgs/by-name/ph/photonvision/package.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, temurin-jre-bin-11
+, bash
+, suitesparse
+, nixosTests
+}:
+
+stdenv.mkDerivation rec {
+  pname = "photonvision";
+  version = "2024.2.3";
+
+  src = lib.getAttr stdenv.hostPlatform.system {
```

Comment:
We probably need an ``or`` here that the code evaluates on other platforms, too.

Reply:
What should the default be, though?

Reply:
I mean something like https://github.com/NixOS/nixpkgs/blob/f7f0117701551541c90db9be172e748bdf241b5a/pkgs/applications/audio/touchosc/default.nix#L54